cms/templates/container.html

```diff
@@ -14,7 +14,7 @@
 )
 from openedx.core.djangolib.markup import HTML, ugettext as _
 %>
-<%block name="title">${xblock.display_name_with_default_escaped} ${xblock_type_display_name(xblock) | h}</%block>
+<%block name="title">${xblock.display_name_with_default | h} ${xblock_type_display_name(xblock) | h}</%block>
```

Comment:
Thanks @mushtaqak.  I actually took care of this in a different PR.

FYI: the instructions for this will soon be available in Read the Docs, and they will have changed slightly for internationalization with ``_()`` and ``HTML()``.